go.mod

```diff
@@ -18,9 +18,9 @@ require (
 	github.com/gogo/protobuf v1.3.1
 	github.com/google/gofuzz v1.1.0
 	github.com/ipfs/go-log v1.0.4
-	github.com/keep-network/keep-common v1.2.1-0.20201112133003-054c43201b4b
+	github.com/keep-network/keep-common v1.2.1-0.20201116121154-016d77aacdce
 	github.com/keep-network/keep-core v1.3.0
-	github.com/keep-network/tbtc v1.1.1-0.20201026093513-cb9246987718
+	github.com/keep-network/tbtc v1.1.1-0.20201116122054-001d3ad0cfc6
```

Comment:
Let's update it to merge commit `986e007`.

Reply:
Done, but I will again change it to latest once [PR](https://github.com/keep-network/tbtc/pull/744) is merged to master